homeassistant/components/kostal_plenticore/const.py

```diff
@@ -40,6 +43,7 @@
             ATTR_UNIT_OF_MEASUREMENT: POWER_WATT,
             ATTR_DEVICE_CLASS: DEVICE_CLASS_POWER,
             ATTR_ENABLED_DEFAULT: True,
+            ATTR_STATE_CLASS: STATE_CLASS_MEASUREMENT,
```

Comment:
We have a new concept to define these sensor descriptions, which integrates with the entity base class. See for example https://github.com/home-assistant/core/pull/53653/files

Reply:
The new description seems nice, but this would lead to a bigger change. I don't know if this is desirable because bigger changes seems to take longer to be merged.

Reply:
I'd also like to have this on the stable release channel as soon as possible so maybe this change could happen in a future PR?

Reply:
We did a release yesterday. The next stable release won't happen until first Wednesday of September.

Reply:
I just don't want to stick with a not working integration for a half year or sth. Waiting a Month isn't that big of an Issue...
Furthermore, aren't there any minor releases for exactly such patches?

Reply:
This refactor can be done in a follow up PR.